### PR TITLE
Release HassOS

### PR DESCRIPTION
**Pull request by pvizeli**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "New Hass.io images"
+description: "We release our new images."
```
Comment (arsaboo) [inline source/_posts/2018-07-11-hassio-images.markdown]: Release of HassOS image

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 month we are happy to announce our new images for Hass.io, based on HassOS.
```
Comment (arsaboo) [inline source/_posts/2018-07-11-hassio-images.markdown]: After 4 months of ______, we are happy to announce our new images for Hass.io, based on HassOS.

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Note that the descriptions of new System can be get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+This will be a bit technical. 
+
+## {% linkable_title Migration %}
+
+The design of HassOS is diferent to ResinOS. That is the reason why we can't provide a OTA update from old too new System.
```
Comment (arsaboo) [inline source/_posts/2018-07-11-hassio-images.markdown]: HassOS is fundamentally different from ResinOS, hence, we cannot provide a OTA update for you to update from Hass.io.

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Make a snapshot of current system. You should remove the bluetooth add-on before, that is not need anymore.
+2. Download the latest [HassOS stable] version.
+3. Flash the images with [Etcher] to SD card.
+5. On Raspberry: if you modify the `config.txt`, you need applay this also on boot partition of HassOS, but __don't__ copy this file!
```
Comment (arsaboo) [inline source/_posts/2018-07-11-hassio-images.markdown]: type - `apply`

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 month we are happy to announce our new images for Hass.io, based on HassOS.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: _After 4 months_

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Note that the descriptions of new System can be get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+We build HassOS on top of the [Buildroot] framework. The focus of the system is to be a very small and high efficent operating system to run Docker like a hypervisor. It have exactly that software installed is need to run a supervisor. We have the focus also on security, there are no default passwords and we use [AppArmor] to protect application and containers on HassOS.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *high efficient* doesn't read well, maybe *highly efficient*?

Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *It have exactly that software installed is need to run a supervisor* maybe instead *It has only the software installed that is needed to run a supervisor*

*We have the focus also on security,* maybe instead *We have also focused on security,*

*protect application and containers* -> *protect the applications and containers*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+Key features:
+- Safe and Secure updating with [Rauc] over USB or internet (OTA)
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: Remove the capital from *Secure* => *secure*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+## {% linkable_title Migration %}
+
+The design of HassOS is diferent to ResinOS. That is the reason why we can't provide a OTA update from old too new System.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *provide a OTA update from old too new System* => *provide an OTA update from from the old Hass.io to the new HassOS system*?

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Make a snapshot of current system. You should remove the bluetooth add-on before, that is not need anymore.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *that is not need anymore* => *as that is not needed anymore*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+2. Download the latest [HassOS stable] version.
+3. Flash the images with [Etcher] to SD card.
+5. On Raspberry: if you modify the `config.txt`, you need applay this also on boot partition of HassOS, but __don't__ copy this file!
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *applay* => *apply*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+4. If you used custom network configuration or ssh development access, we can use a [USB stick]. You can simple copy the resin-sample into `network` folder on USB stick.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *If you used custom network configuration or ssh development access, we can use a* => *If you used a custom network configuration or ssh development access, you can use a*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+5. Take SD card into device slot and boot your system like before.
+6. Copy the snapshot into host with SSH or samba add-on.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *Copy the snapshot into host* => *Copy the snapshot to the host*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+7. Restore it over the Hass.io panel
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *Restore it over the* => *Restore it using the*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+
+## {% linkable_title Migration %}
+
+The design of HassOS is diferent to ResinOS. That is the reason why we can't provide a OTA update from old too new System.
+
+1. Make a snapshot of current system. You should remove the bluetooth add-on before, that is not need anymore.
+2. Download the latest [HassOS stable] version.
+3. Flash the images with [Etcher] to SD card.
+5. On Raspberry: if you modify the `config.txt`, you need apply this also on boot partition of HassOS, but __don't__ copy this file!
+4. If you used custom network configuration or ssh development access, we can use a [USB stick]. You can simple copy the resin-sample into `network` folder on USB stick.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *If you used custom network configuration or ssh development access, we can* => *If you used a custom network configuration, or SSH development access, you can*

Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *sample into `network` folder on USB stick* => *sample into the `network` folder on the USB stick*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+5. Take SD card into device slot and boot your system like before.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *Take SD card into device slot* => *Put the SD card into the device slot*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+6. Copy the snapshot into host with SSH or samba add-on.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *Copy the snapshot into host with* => *Copy the snapshot to the host with the*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+7. Restore it over the Hass.io panel
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *Restore it over the* => *Restore it using the*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 18]
```diff
@@ -0,0 +1,60 @@
...
+
+## {% linkable_title Future %}
+
+We have now a wounderful base system and can now work to integrate all this features into Hass.io and to UI. So we plan to integrate the Network configuration and bluetooth into UI. The goal is a full featured hub system they allow anybody to use Home Assistant.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *We have now a wounderful base system and can now work to integrate all this features into Hass.io and to UI* => We have now a wonderful base system and can now work to integrate all these features into Hass.io and the UI*

Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *So we plan to integrate the Network configuration and bluetooth into UI* => *We plan to integrate the network configuration and Bluetooth into the UI*

Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *The goal is a full featured hub system they allow anybody to use Home Assistant.* => *The goal is a fully featured hub system that allows anybody to use Home Assistant.*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 19]
```diff
@@ -0,0 +1,60 @@
...
+
+We would also adapt more Hass.io API function into Home Assistant i.e. to monitor the system usage of a Add-on or Home Assistant container.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *We would also adapt more Hass.io API function* => *We will also adapt more Hass.io API functions*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 20]
```diff
@@ -0,0 +1,60 @@
...
+
+Thanks a lot for all the hardware donations. We start now to porting HassOS to all this hardware and try to support an new device ever 7-14 days until we support all IoT devices they we reached.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *We start now to porting* => *We'll now start porting*

*support an new device ever* => *support a new device every*

*they we reached* => ? *that we received*?

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 21]
```diff
@@ -0,0 +1,60 @@
...
+
+Note that the descriptions of new System can be get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+We build HassOS on top of the [Buildroot] framework. The focus of the system is to be a very small and high efficent operating system to run Docker like a hypervisor. It have exactly that software installed is need to run a supervisor. We have the focus also on security, there are no default passwords and we use [AppArmor] to protect application and containers on HassOS.
+
+Key features:
+- Safe and Secure updating with [Rauc] over USB or internet (OTA)
+- Use LZ4 compressed root files and parts of Memory
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: Remove capital from Memory

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 22]
```diff
@@ -0,0 +1,60 @@
...
+- Read-Only root file system, designed to run on embedded systems
+- Dbus connected hosts services
+- Latest LT linux Kernel
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: Maybe expand LT (I'm guessing that's long term support?). Remove the capital from Kernel

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 23]
```diff
@@ -0,0 +1,60 @@
...
+- Latest Docker-CE version
+- Full supported NetworkManager and Bluez
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *Full supported* => *Fully support*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 24]
```diff
@@ -0,0 +1,60 @@
...
+
+## {% linkable_title Migration %}
+
+The design of HassOS is diferent to ResinOS. That is the reason why we can't provide a OTA update from old too new System.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *diferent* => *different*

*provide a OTA update from old too new System* => *provide an OTA update to HassOS from an old ResinOS system*

**Comment by DubhAd** [comment]
Sorry, I appreciate this is a bit "death by a thousand comments"

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 25]
```diff
@@ -0,0 +1,67 @@
...
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 months we are happy to announce our new images for Hass.io, based on HassOS.
+
+Note that the descriptions of the new system can get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+We have build HassOS on top of the [Buildroot] framework. The focus of the system is to be a very small and high efficient operating system to run Docker like a hypervisor. It has installed just enough software to run a supervisor. We have also focussed on security, there are, for example, no default passwords and we use [AppArmor] to protect the applications and containers on HassOS.
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: *We have build* => *We have built*

*and high efficient* => *and highly efficient*

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 26]
```diff
@@ -0,0 +1,67 @@
...
+
+- Safe and secure updating with [Rauc] over USB or internet (OTA)
+- Uses an LZ4 compressed root filesystem and parts of the memory
+- Read-only root filesystem, designed to run on embedded systems
+- Dbus connected hosts services
+- Latest LT Linux kernel
+- Latest Docker-ce version
+- Fully supported NetworkManager
+- Bluetooth support using Bluez
+- Supports lots different Hardware
```
Comment (DubhAd) [inline source/_posts/2018-07-11-hassio-images.markdown]: Remove the capital from Hardware

File: source/getting-started/index.markdown [hunk 27]
```diff
@@ -25,34 +25,35 @@ We will need a few things to get started with installing Home Assistant. Links b
 - [Micro SD Card](http://a.co/gslOydD). Get one that is Class 10 as they are more reliable. Size 32 GB or bigger recommended.
 - SD Card reader. Part of most laptops, and also available as [standalone USB sticks](http://a.co/5FCyb0N) (the brand doesn't matter, just pick the cheapest)
 - Ethernet cable (optional, Hass.io can work with WiFi too)
+- A USB-Stick (optional, to configuration unattendend)
```
Comment (DubhAd) [inline source/getting-started/index.markdown]: *to configuration unattendend* maybe should be *for unattended configuration*?

File: source/getting-started/index.markdown [hunk 28]
```diff
@@ -25,34 +25,35 @@ We will need a few things to get started with installing Home Assistant. Links b
...
-[pi3]: https://github.com/home-assistant/hassio-build/releases/download/1.3/resinos-hassio-1.3-raspberrypi3.img.bz2
+[pi3-32]: https://github.com/home-assistant/hassos/releases/download/1.5/hassos_rpi3-1.5.img.gz
+[pi3-64]: https://github.com/home-assistant/hassos/releases/download/1.5/hassos_rpi3-64-1.5.img.gz
 
 ### {% linkable_title Installing Hass.io %}
 
 1. Put the SD card in your SD card reader.
 1. Open Etcher, select the Hass.io image and flash it to the SD card.
-1. WiFi setup only: open the file `system-connections/resin-sample` (from the `resin-boot` volume on the SD card) with a text editor. Change `ssid` to be your network name and `psk` to be your password. Note: When connecting to a 5 GHz-only network, make sure your device supports that. If you don't know what that means, then you probably have nothing to worry about.
+1. WiFi setup only: Format a USB-Stick with name `CONFIG` and copy one of [the example] to the file `network/hassos-wifi`.
```
Comment (DubhAd) [inline source/getting-started/index.markdown]: `[the example]` => `[the examples]`

File: source/getting-started/index.markdown [hunk 29]
```diff
@@ -25,34 +25,35 @@ We will need a few things to get started with installing Home Assistant. Links b
...
 1. Unmount the SD card and remove it from your SD card reader.
 1. Insert the SD card into your Raspberry Pi 3. If you are going to use an Ethernet cable, connect that too.
 1. Connect your Raspberry Pi to the power supply so it turns on.
-1. The Raspberry Pi will now boot up, connect to the Internet and download the latest version of Home Assistant. This will take about 20 minutes. Do not expect anything but the Home Assistant logo on the HDMI display, or any use for a keyboard or mouse.
+1. The Raspberry Pi will now boot up, connect to the Internet and download the latest version of Home Assistant. This will take about 20 minutes.
 1. Home Assistant will be available at [http://hassio.local:8123][local]. Shortly after the download has started, a simple preparation status page will be available at this URL. Point your browser there. The page refreshes automatically, and Home Assistant will be shown when the download is complete.
 
 [local]: http://hassio.local:8123
+[the example]: https://github.com/home-assistant/hassos/blob/dev/Documentation/network.md
```
Comment (DubhAd) [inline source/getting-started/index.markdown]: `[the example]` => `[the examples]`

**Review by DubhAd — COMMENTED** [review]
A few more language changes

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 30]
```diff
@@ -0,0 +1,67 @@
...
+
+## {% linkable_title Migration %}
+
+The design of HassOS is different from ResinOS. Because of this, we can't provide an OTA update from the old (ResinOS) too new system (HassOS).
+
+You need to perform the following steps to upgrade:
+
+1. If you have installed the Bluetooth add-on, please remove it, since it is not required anymore.
+2. Make a Hass.io snapshot of your current system and download it to your computer.
+3. Download the latest [HassOS stable] version.
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: Should be `Hass.io`

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 31]
```diff
@@ -0,0 +1,67 @@
...
+4. Flash the downloaded HassOS image with [Etcher] to your SD card.
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: `Hass.io`

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 32]
```diff
@@ -0,0 +1,67 @@
...
+5. Raspberry Pi: In case you have modified the `config.txt` (in the boot partition), you will also need to apply these changes to HassOS. Do **NOT** simply copy from the file from your old setup into HassOS! Apply those changes manually!
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: How can a user apply those changes? Link to instructions ?

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 33]
```diff
@@ -0,0 +1,67 @@
...
+6. If you used custom network configuration or had configured SSH development access, you need to use a [USB stick]. Copy the resin-sample into `network` folder on a USB stick and insert it into your device.
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: `use a [USB stick]` -> `create a [Configuration USB stick]`

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 34]
```diff
@@ -0,0 +1,67 @@
...
+- Bluetooth support using Bluez
+- Supports lots different Hardware
+
+## {% linkable_title Migration %}
+
+The design of HassOS is different from ResinOS. Because of this, we can't provide an OTA update from the old (ResinOS) too new system (HassOS).
+
+You need to perform the following steps to upgrade:
+
+1. If you have installed the Bluetooth add-on, please remove it, since it is not required anymore.
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: What happens if they keep it enabled, will the snapshot be able to be restored?

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 25 above)
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: `focused`

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 35]
```diff
@@ -0,0 +1,67 @@
...
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 months we are happy to announce our new images for Hass.io, based on HassOS.
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: Add some more text:

HassOS is a brand new operating system that we have created specifically for the purpose to run Hass.io.

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 36]
```diff
@@ -0,0 +1,67 @@
...
+
+Note that the descriptions of the new system can get quite technical. Feel free to jump to the [migration steps].
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: Let's remove this paragraph.

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 25 above)
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: just enough software installed

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 37]
```diff
@@ -0,0 +1,67 @@
...
+- Dbus connected hosts services
+- Latest LT Linux kernel
+- Latest Docker-ce version
+- Fully supported NetworkManager
+- Bluetooth support using Bluez
+- Supports lots different Hardware
+
+## {% linkable_title Migration %}
+
+The design of HassOS is different from ResinOS. Because of this, we can't provide an OTA update from the old (ResinOS) too new system (HassOS).
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: we can't provide an OTA update from the old ResinOS system to the new HassOS based system.

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 25 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: ✔️

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 35 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: ✔️

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 36 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: Agreed. ✔️

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 25 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: ✔️

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 37 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: ✔️

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 30 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: ✔️

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 31 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: ✔️

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 32 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: Agree, unclear to me as well.
Those are actually kinda undocumented changes (maybe they are documented at the Raspberry Pi website?).

Nevertheless, a user who has modified this is most likely well aware.

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 33 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: ✔️

File: source/_posts/2018-07-11-hassio-images.markdown (hunk 34 above)
Comment (frenck) [reply source/_posts/2018-07-11-hassio-images.markdown]: The end of the world?

No sure, I'll guess it simply doesn't work, but still will boot. So removing it afterward?
@pvizeli Could you confirm?

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 38]
```diff
@@ -0,0 +1,64 @@
...
+
+- Safe and secure updating with [Rauc] over USB or internet (OTA)
+- Uses an LZ4 compressed root filesystem and parts of the memory
+- Read-only root filesystem, designed to run on embedded systems
+- Dbus connected hosts services
+- Latest LT Linux kernel
+- Latest Docker-ce version
+- Fully supported NetworkManager
+- Bluetooth support using Bluez
+- Supports lots different hardware
```
Comment (MartinHjelmare) [inline source/_posts/2018-07-11-hassio-images.markdown]: lots of different

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 39]
```diff
@@ -0,0 +1,64 @@
...
+The design of HassOS is different from ResinOS. Because of this, we can't provide an OTA update from the old ResinOS system to the new HassOS based system.
+
+You need to perform the following steps to upgrade:
+
+1. If you have installed the Bluetooth add-on, please remove it, since it is no longer required.
+2. Make a Hass.io snapshot of your current system and download it to your computer.
+3. Download the latest [Hass.io stable][installation] version.
+4. Flash the downloaded Hass.io image with [Etcher] to your SD card.
+5. Raspberry Pi: In case you have modified the `config.txt` (in the boot partition), you will also need to apply these changes to HassOS. Do **NOT** simply copy the file from your old setup into HassOS! Apply those changes manually!
+6. If you use a custom network configuration or have configured SSH development access, you need to create a configuration [USB stick]. Copy the resin-sample into `network` folder on a USB stick and insert it into your device.
```
Comment (MartinHjelmare) [inline source/_posts/2018-07-11-hassio-images.markdown]: Copy the resin-sample into the `network` folder

File: source/_posts/2018-07-11-hassio-images.markdown [hunk 40]
```diff
@@ -0,0 +1,64 @@
...
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 months of hard work, we are happy to announce our new images for Hass.io, based on HassOS. HassOS is a brand new operating system that we have created specifically for the purpose of running Hass.io.
```
Comment (balloob) [inline source/_posts/2018-07-11-hassio-images.markdown]: Let's add one more sentence: And yes, this supports the Raspberry Pi 3 B+!

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
@pvizeli Thanks for all your hard work!

bitmoji

**Pull request merged**